github_cog.py

```diff
@@ -0,0 +1,111 @@
+import discord
+import github.GithubException
+from discord.ext import commands
+from github import Github
+from contextlib import closing
+import studybot
+
+
+# noinspection PyBroadException
```

Comment:
Don't add this to the top of a class. Add it on each line you must have a broad exception. As a rule if you have a bunch of broad exceptions try doing it another way.

Reply:
I didn't even realize this was added, I was just trying to get pycharm to shut up.